### ISSUE TITLE
Drop Reasons

### ISSUE THREAD

**Issue by eastzone** [Broadcom-Switch/BroadView-Instrumentation#1]
I see that the drop reasons are mentioned in the following places. However, I don't see how it's implemented in OpenNSL API. In fact, I cannot even find them in Broadcom SDK. Are they available publicly in OpenNSL/Broadcom SDK yet? Just make sure I am not missing anything.

https://github.com/Broadcom-Switch/BroadView-Instrumentation/blob/master/src/public/packet_trace.h#L49

https://github.com/Broadcom-Switch/BroadView-Instrumentation/blob/master/src/sb_plugin/sb_brcm/sb_common/packet_trace/sbplugin_common_packet_trace.c#L439

**Comment by simonknee** [comment]
You are correct, these features are not functional in the current software release  They are there as placeholders for future releases.
